docs/README.md

```diff
@@ -68,14 +68,61 @@ macOS (OS X) development environment.
 # User documentation
 
 * [Getting started (How-To with examples)](usage.md)
+
+## C/C++ Analysis
 * [Analyzer User guide](analyzer/user_guide.md)
-* [Webserver User guide](web/user_guide.md)
-* [Requiring credentials to view analysis results](web/authentication.md)
-* [Overview about connecting multiple analysis run databases](web/products.md)
-* [Permission management](web/permissions.md)
-* [Usage of PostgreSQL database](web/postgresql_setup.md)
-* [How to deal with false positives](analyzer/false_positives.md)
-* [Store Cppcheck analyzer reports](/tools/report-converter/README.md#cppcheck)
+* [Avoiding or suppressing false positives](analyzer/false_positives.md)
+* [Checker and Static Analyzer configuration](analyzer/checker_and_analyzer_configuration.md)
+* [Checker documentation](web/checker_docs.md)
+* [GCC incompatibilities](analyzer/gcc_incompatibilities.md)
+
+## Web based report management
+* [Webserver User Guide](web/user_guide.md)
+* [Command line and WEB UI Feature overview](feature_comparison.md)
+* [Configuring Authentication](web/authentication.md)
+* [Configuring Authorization](web/permissions.md)
+* [Configuring Server Logging](logging.md)
+* [Deploy server using docker](web/docker.md#deployment)
+* [Setting up CI gating with Gerrit and Jenkins](jenkins_gerrit_integration.md)
+* [PostgreSQL database backend setup guide](web/postgresql_setup.md)
+* [CodeChecker server and database schema upgrade guide](web/db_schema_guide.md)
+* [Setting up multiple CodeChecker repositories in one server](web/products.md)
+
+### Storage of reports from analyzer tools
+CodeChecker can be used as a generic tool for visualizing analyzer results.
+
+The following tools are supported:
+
+| Language       | Analyzer     |
+| -------------- |--------------|
+| **C/C++**      | [Clang Tidy](https://clang.llvm.org/extra/clang-tidy/)  |
+|                | [Clang Static Analyzer](https://clang-analyzer.llvm.org/)    |
+|                | [Cppcheck](/tools/report-converter/README.md#cppcheck)    |
+|                | [Clang Sanitizers](supported_code_analyzers.md#clang-sanitizers)    |
+|                | [Facebook Infer](/tools/report-converter/README.md#fbinfer)    |
+| **Java**       | [SpotBugs](/tools/report-converter/README.md#spotbugs)    |
+|                | [Facebook Infer](/tools/report-converter/README.md#fbinfer)    |
+| **Python**     | [Pylint](/tools/report-converter/README.md#pylint)    |
+|                | [Pyflakes](/tools/report-converter/README.md#pyflakes)    |
+| **JavaScript** | [ESLint](/tools/report-converter/README.md#eslint)    |
+| **TypeScript** | [TSLint](/tools/report-converter/README.md#tslint)    |
+| **Go**         | [Golint](/tools/report-converter/README.md#golint)    |
```

Comment:
Maybe also sort the languages (we can keep C/C++ on top)

Reply:
i think it is okay to sort them like this. Order of support added.